bin/flutter.bat

```diff
@@ -93,6 +93,7 @@ GOTO :after_subroutine
     )
 
   :do_snapshot
+    IF EXIST "%FLUTTER_ROOT%\version" DEL %FLUTTER_ROOT%\version
```

Comment:
Quotes around the argument to DEL, just in case there a spaces in the FLUTTER_ROOT path?

Reply:
done